gosnappi/common.go

```diff
@@ -116,31 +115,28 @@ func (obj *httpTransport) SetVerify(value bool) HttpTransport {
 	return obj
 }
 
-type api struct {
+type apiSt struct {
```

Comment:
maybe `apiImpl`, since `apiSt` sounds like api status ?

Can be addressed after merge.